ReactiveObjCTests/NSObjectRACSelectorSignalSpec.m

```diff
@@ -194,6 +196,42 @@ - (id)objectValue;
 		expect(@([object respondsToSelector:selector])).to(beTruthy());
 	});
 
+	qck_it(@"hotpatch with dynamic subclass: disable a selector then re-implementing it in forwardInvocation:", ^{
+		__block BOOL patchApplied = NO;
+		
+		RACTestObject *object = [[RACTestObject alloc] init];
+		
+		// A dynamic subclass created.
+		[[RACObserve(object, objectValue) publish] connect];
+		[object rac_signalForSelector:@selector(lifeIsGood:)];
+		
+		// Simulator of hotpatch:
+		SEL selectorPatched = @selector(methodHotpatch);
```

Comment:
Mind to replicate the test case for the scenario of the patched method being the same as the one being intercepted via `rac_signalForSelector` (i.e. `lifeIsGood:`)?